### DISCUSSION TITLE
Save screenshots of models by code?

### DISCUSSION THREAD

**Discussion by afunrwth** [Dlubal-Software/RSTAB_Python_Client#9] [Q&A] [answered]
Is it possible to save screenshots of models by code?

**Comment by OndraMichal** [comment, marked as answer]
Hi @afunrwth,
there are no fucntions to save the screenshots, but I'm sure you can find some tutorials online like this one on [geegksforgeeks.org](https://www.geeksforgeeks.org/how-to-take-screenshots-using-python/).
